### ISSUE TITLE
Adobe Accounts - Updating payment details is broken by blocks

### ISSUE THREAD

**Issue by jamesjenner** [uBlockOrigin/uAssets#158]
Replace the bracketed [...] placeholders with your own information.
### URL(s) where the issue occurs

`https://accounts.adobe.com/plans/7CA4511156D93FE40A744C55/edit-payment#`
### Describe the issue

When editing payment details in the accounts section for adobe, the changes cannot be saved. There is no visual indicator other than clicking the save button has no action. As such, I'm not including a screen shot (as there is nothing to demonstrate).

Looking at the console log, I can see two errors:
Uncaught ReferenceError: AppMeasurement is not defined - production.js:717
POST https://www.adobe.com/dynaTraceMonitor?dtCookie=B.... net::ERR_BLOCKED_BY_CLIENT dtagent628_bxjpr23t_1019.js:26

If I turn off the blocking then it works. 
### Screenshot(s)

None as there is nothing to see visually.
### Versions
- Browser/version: Chrome 53.0.2785.143 m
- uBlock Origin version: 1.9.10
### Settings
- left as default (AFAICT)

Filters that are applied are:
- uBlock filters
- uBlock filters - Badware risks
- uBlock filters - Privacy
- uBlock filters - Unbreak

Ads
- Easylist
- Peter Lowe's Ad server list

Malware domains
- Malware Domain List
- Malware domains
### Notes

An adobe account will be required to test this. Please send me a message if you would like me to test any changes.
I identified the problem via the console for chrome. Found several errors and then turned off filtering to test if it resolved the issue. It did.
I looked at all the filters that are selected and searched for a reference to adobe.com. Found referneces in two lists: 
- EasyList 
  - get.adobe.com###bannerDisplay 
  - adobe.com##.SiteFooterRow[style="font-size:9px;font-family:Arial"]
  - adobe.com##.TextSmall[align="center"][style="font-size:9px;font-family:Arial"]
- EasyPrivacy
  - ||adobe.com^*/omniture_s_code.js

**Comment by gorhill** [comment]
> `||adobe.com^*/omniture_s_code.js`

Did you try to counter the block with a rule or an exception filter?

**Comment by jamesjenner** [comment]
I just turned off blocking for the page and it fixed the issue. I'm not familar with adding rules or exception filters. When I turned off blocking, it added `accounts.adobe.com` to the white list. This fixed the problem.

I'm not familiar with regex (I presume that's how the filters are applied), so I'm not sure which reference is 
causing the problem (or if it's some other mechanism that I'm not aware of). 

I can add an exception to test if you like, but you will have to explain the pattern to apply and where it should be applied (I presume "My filters").
